packages/contracts-bedrock/src/L2/CrossL2Inbox.sol

```diff
@@ -64,6 +78,27 @@ contract CrossL2Inbox is ICrossL2Inbox, ISemver, TransientReentrancyAware {
     /// @param id Encoded Identifier of the message.
     event ExecutingMessage(bytes32 indexed msgHash, Identifier id);
 
+    function setInteropStart() external {
+        // Check that caller is the DEPOSITOR_ACCOUNT
```

Comment:
This is a good mix of solidity + yul, since this function is only called once it doesn't need to be all yul, making it easy to read is preferred when its only called once